train.py

```diff
@@ -28,7 +29,7 @@ def parse_args():
     parser.add_argument('--epoch', dest='epoch', help='epoch of pretrained model',
                         default=1, type=int)
     parser.add_argument('--prefix', dest='prefix', help='new model prefix',
-                        default=os.path.join(os.getcwd(), 'model', 'ssd'), type=str)
+                        default=os.path.join(os.getcwd(), 'output', 'exp1', 'ssd'), type=str)
```

Comment:
OSError: [Errno 2] No such file or directory: '/home/ubuntu/test/mxnet-ssd/output/exp1' if not explicitly created for this.

Reply:
I can erase that, or automatically create this folder.. what do you propose? 
I find it more convenient to have an 'output' folder for all experiments checkpoints, ROC graphs, detection vis and logs to tensorboard